chargeio/services/Merchant/MerchantService.cs

```diff
@@ -1,46 +1,37 @@
-﻿using ChargeIO.Infrastructure;
+﻿using ChargeIo.services;
+using ChargeIO.Infrastructure;
 using ChargeIO.Models;
 
 namespace ChargeIO.Services.Merchant
 {
-	public class MerchantService
-	{
-        private string SecretKey { get; }
-        
-        public MerchantService(string secretKey = "")
+    public sealed class MerchantService : ServiceBase

```

Comment:
We shouldn't be sealing our classes -- we shouldn't be restricting users from inheriting from our classes.